hack/k8s/flood-pipeline.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: extensions/v1beta1
```

Comment:
This should not be here..

Reply:
yeah I'd considered that too. I implicitly put that one together with the bag of unused files.

Reply:
see #432